examples/osgi-test-example-bnd/cnf/ext/defaults.bnd

```diff
@@ -0,0 +1,40 @@
+java: java
+javac: javac
+javac.source: 1.8
+javac.target: 1.8
+
+# This is the version of JUnit that will be used at build time and runtime
+osgitest: org.osgi.test.junit5;version=latest,\
+	org.osgi.test.junit5.cm;version=latest,\
+	org.osgi.test.common;version=latest,\
+	org.osgi.test.assertj.framework;version=latest
+
+# This is the version of JUnit that will be used at build time and runtime
+junit: junit-jupiter-api;version="[5.7.0,6)",\
```

Comment:
Why version ranges here? Just use latest since you control the version in the repo.

Reply:
changes to latest